src/styles/_catalog.scss

```diff
@@ -1,14 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1,200px);
```

Comment:
since you use this value many times you could create a variable for it! and store it in a variables file.